docs/content/tutorials/index.md

```diff
@@ -116,21 +117,13 @@ All persistent state such as the cluster metadata store and segments for the ser
 
 Later on, if you'd like to stop the services, CTRL-C to exit the `bin/start-micro-quickstart` script, which will terminate the Druid processes.
 
-### Resetting cluster state
-
-If you want a clean start after stopping the services, delete the `var` directory and run the `bin/start-micro-quickstart` script again.
-
-Once every service has started, you are now ready to load data.
+Once the cluster has started you can navigate to [http://localhost:8888](http://localhost:8888).
+This is the router node that also serves the Druid console.
```

Comment:
The [Druid router process](link to docs), which serves the Druid console, resides at this address.